pandas/core/dtypes/dtypes.py

```diff
@@ -1291,7 +1290,7 @@ class PandasDtype(ExtensionDtype):
 
     _metadata = ("_dtype",)
 
-    def __init__(self, dtype: NpDtype | PandasDtype | None):
+    def __init__(self, dtype: str | np.dtype | PandasDtype | None):
```

Comment:
why this change?  what prevents me from passing `int`?

Reply:
Fails mypy for some reason.  Might have to do with the typing of how the function `np.dtype` is typed by numpy

Reply:
`PandasDtype(int)` returns PandasDtype('int64')

pls let the question-asker do the "mark as resolved"

Reply:
does npt.DTypeLike work?